### DISCUSSION TITLE
为什么vps面板总流量消耗是xui面板上下行总流量的两倍

### DISCUSSION THREAD

**Discussion by osongsongo** [FranzKafkaYu/x-ui#252] [General]
比如xui显示上下行各消耗1g，总计消耗2g。但是vps的面板上显示的上下行消耗总和是4g。是我设置错误了吗？还是原理就是这样？

**Comment by FranzKafkaYu** [comment]
你的数据->经过服务器(代理入站，计算一次)-目标网站>(代理出站，计算一次)，这是上行，服务器流量双向计算，所以是2倍;下行也是类似。
